test/run-tests

```diff
@@ -467,7 +497,15 @@ class Task:
             # used to be tested by running all playbooks `test/test_*.yml`.
             # Support both, but prefer the standard way. Can be removed once
             # all repos are moved over.
-            playbookglob = f"{sourcedir}/tests/tests*.yml"
+            if test_collections:
+                playbookglob = (
+                    os.environ.get("COLLECTION_TESTS_DEST_PATH")
```

Comment:
use `os.environ["COLLECTION_TESTS_DEST_PATH"]` here to catch bugs - if it is not set it will raise an error instead of silently use `/tests/..` as the path.